release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -621,6 +621,91 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
 [id="ocp-4-11-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id=ocp-4-11-monitoring-updates-to-monitoring-stack-components-and-dependencies]
+==== Updates to monitoring stack components and dependencies
+
+Updates to versions of monitoring stack components and dependencies include the following:
+
+* Alertmanager to 0.24.0
+* kube-state-metrics to 2.5.0
+* Prometheus to 2.36.2
+* Prometheus operator to 0.57.0
+* Thanos to 0.26.0
+
+[id=ocp-4-11-monitoring-changes-to-alerting-rules]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `KubePersistentVolumeInodesFillingUp` alert, which functions similarly to the existing `KubePersistentVolumeFillingUp` alert but is applied to inodes rather than volume space.
+** Added the `PrometheusScrapeBodySizeLimitHit` alert to detect targets hitting the body size limit.
+** Added the `PrometheusScrapeSampleLimitHit` alert to detect targets hitting the sample limit.
+
+* *Changed*
+** Fixed the `KubeDaemonSetRolloutStuck` alert to use the updated metric `kube_daemonset_status_updated_number_scheduled` from `kube-state-metrics`.
+** Replaced the `KubeJobCompletion` alert with `KubeJobNotCompleted`. The new `KubeJobNotCompleted` alert avoids false positives when an earlier job failed but the most recent job succeeded.
+** Updated the `NodeNetworkInterfaceFlapping` alert to exclude `tunbr` interfaces from the alert expression.
+
+[id=ocp-4-11-monitoring-enable-alert-routing-for-user-workload-momitoring]
+==== Enable alert routing for user workload monitoring
+A cluster administrator can now enable alert routing for user workload monitoring so that developers and other users can configure custom alerts and alert routing for their user-defined projects.
+
+[id=ocp-4-11-monitoring-enable-dedicated-alertmanager-for-user-defined-alerts]
+==== Enable a dedicated Alertmanager instance for user-defined alerts
+You can optionally enable a separate instance of Alertmanager dedicated to sending alerts only for user-defined projects.
+This feature can help reduce the load on the default platform Alertmanager instance and can better separate user-defined alerts from default platform alerts.
+
+[id=ocp-4-11-monitoring-use-additional-authenticaion-settings-for-remote-write]
+==== Use additional authentication settings for remote write configuration
+You can now use the following authentication methods to access a remote write endpoint: AWS Signature Version 4, custom Authorization header, and OAuth 2.0. 
+Before this release, you could only use TLS client and basic authentication.
```

Comment:
Minimalism 

Is this last sentence necessary? I do like it because it demonstrates the advancement between versions. However, most of you other sections don't state `Before this release...` So I'm wondering why you included this information here? If no particular reason, perhaps it could be removed since it goes to figure that the new release brings advancement and changes that previous versions did not have. 

Also, I would think that users would know what they currently were limited by with the older version. Nonetheless, I do think this sentence does a nice job stating the value-add with this release. So it's kind of a toss up!